extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/VolumeItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.kubernetes.deployment;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class VolumeItem {
```

Comment:
I would name it `VolumeConfig` to align with the rest of the code.

Reply:
I guess you mean VolumeItemConfig, right ?

Reply:
`VolumeItemConfig` or `VolumeConfig` both are fine.